modules/react/layout/lib/Stack.tsx

```diff
@@ -6,8 +6,22 @@ import {Flex, FlexProps} from './Flex';
 import {getValidChildren} from './utils/getValidChildren';
 import {stack, StackStyleProps} from './utils/stack';
 
+/**
+ * ### ⚠️ Stack has been soft-deprecated and will be removed in v9 ⚠️
+ * - If you would like resolve this now, run this codemod: `npx @workday/canvas-kit-codemod <transform> [path]`
```

Comment:
```suggestion
 * - If you would like resolve this now, run this codemod: `npx @workday/canvas-kit-codemod softDeprecate/Stack [path]`
```

Reply:
we should give them the codemod to run yeah?

Reply:
Thanks for the suggestion. 

This was going to be added once our file structure was agreed on. 

Forgot to add that back in and it's now updated.